src/main/java/com/mraof/minestuck/entity/consort/ConsortEntity.java

```diff
@@ -77,6 +80,7 @@ public class ConsortEntity extends AnimatedPathfinderMob implements MenuProvider
 	
 	private final AnimatableInstanceCache cache = GeckoLibUtil.createInstanceCache(this);
 	private final EnumConsort consortType;
+	private Dialogue dialogue;
```

Comment:
Since dialogue can be reloaded mid-game, this should likely not be storing a reference to the dialogue object directly. I suggest storing the dialogue path instead.